common/src/main/java/io/druid/concurrent/Execs.java

```diff
@@ -55,6 +55,11 @@ public static ThreadFactory makeThreadFactory(String nameFormat)
     return new ThreadFactoryBuilder().setDaemon(true).setNameFormat(nameFormat).build();
   }
 
+  public static ExecutorService threadPool(String nameFormat)
```

Comment:
I think this name should be more specific about what it actually is: cachedThreadPool? unboundedThreadPool?